### ISSUE TITLE
bindings should be build automagically as part of "pub get"

### ISSUE THREAD

**Issue by muth02446** [google/dart-gl#12]
This will make it much likely that people with use this library

**Comment by jtmcdole** [comment]
Yep, I'd like the bindings to be built with Dart instead of python as well, reducing the number of dependencies.

**Comment by jtmcdole** [comment]
We're not building with pub get yet - but that'll be next on the list.
